wrappers/nodejs/scripts/generate-dist-package.js

```diff
@@ -0,0 +1,50 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+//   Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+// Purpose: pack librealsense dir & wrappers/nodejs dir into npm package
+
+const {exec} = require('child_process');
+const package = require('../package.json');
+const jf = require('jsonfile');
+
+function genDistPackage() {
+  genPackageJson().then(() => {
+    callDistScript();
+  });
+}
+
+function genPackageJson() {
+  return new Promise((resolve, reject) => {
+    exec('cp -f ./package.json scripts/npm_dist/package.json', (error, stdout, stderr) => {
```

Comment:
How about use existing node module or `https://stackoverflow.com/questions/11293857/fastest-way-to-copy-file-in-node-js` instead of exec?

Reply:
just a small file... the fastest way won't get any real benefit.

Reply:
the concern is about `cp -r` is cross platform or not. We may have future platform to support. Anyway, you can decide to keep or change, I'm both fine.